doc/Makefile

```diff
@@ -92,7 +92,7 @@ MANPAGES_3_DUMMY_EXP =
 MANPAGES_1_DUMMY_EXP =
 
 # libpmem2
-MANPAGES_7_MD_PMEM2 = libpmem2/libpmem2.7.md
+MANPAGES_7_MD_PMEM2 = libpmem2/libpmem2.7.md libpmem2/libpmem2_unsafe_shutdown.7.md
```

Comment:
1. make sure how it works on pmem.io. Look here: https://github.com/pmem/pmdk/tree/gh-pages/_layouts
(7  was used only for libraries so it can be treated in a different way than intended )
2. Add support here: https://github.com/pmem/pmdk/tree/gh-pages/_data